sim/pokemon.js

```diff
@@ -89,6 +89,10 @@ class Pokemon {
 		/**@type {string | boolean} */
 		this.moveThisTurn = '';
 
+		// For Stomping Tantrum
+		this.moveLastTurnResult = undefined;
+		this.moveThisTurnResult = undefined;
```

Comment:
In which case, we definitely don't need both these variables. You just need `this.lastMoveResult`, and exclude Dancer and Instruct moves from overwriting it.

Reply:
Oh, wait, never mind. In hindsight, it does seem to be tracking multiple results.